### ISSUE TITLE
Rename "(Remote)" to "(Federated Share)" in autocomplete dialog

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#29015]
@hodyroff @pmaier1 to discuss, maybe need shorter wording ?

**Comment by hodyroff** [comment]
Just "(Federated)" should work, IMHO. Admin can switch it off if deemed to complex for his users. Can it be switched on/off by group currently?

**Comment by PVince81** [comment]
Federated sharing can only be disabled globally, there are only two options:
- disable outgoing shares, which affects this autocomplete result
- disable incoming shares

**Comment by pmaier1** [comment]
I'm in favor of "(Federated)". Not sure if there is any better wording but as we call the feature Federation it should fit. Some wording that would express "you're sharing to another instance" would make it too complex for people, I think. If it would already work perfectly I would even say we remove it and let people think it would be a "normal" share (the Dropbox approach to things.. only tell people what they really need to know to avoid confusion). Anyway, "(Federated)" is a lot easier to understand than "(Remote)" as it's less technical language.

**Comment by PVince81** [comment]
let's promote this as junior job if anyone at the conf wants to get started with our dev process 😄

**Comment by PVince81** [comment]
PR here https://github.com/owncloud/core/pull/29166

**Comment by ownclouders** [comment]
Hey, this issue has been closed because the label `status/STALE` is set and there were no updates for 7 days. Feel free to reopen this issue if you deem it appropriate.

**Comment by PVince81** [comment]
reopening, we do want this

**Comment by PVince81** [comment]
PR here https://github.com/owncloud/core/pull/29995

**Comment by sharidas** [comment]
Guess we can close this as backport https://github.com/owncloud/core/pull/30046  and https://github.com/owncloud/core/pull/29995 is merged

**Comment by davitol** [comment]
![screen shot 2018-01-17 at 12 48 55](https://user-images.githubusercontent.com/10596560/35042654-d4be4e6e-fb89-11e7-98d6-807a64bcae8e.png)
We should change the remote wording not only in autocomplete dialog but also when the share is listed below

**Comment by PVince81** [comment]
here https://github.com/owncloud/core/pull/30171
